tests/test_pdf.py

```diff
@@ -1,8 +1,29 @@
+import re
+from pathlib import Path
+from typing import List
+
 import pytest
 
 
 @pytest.mark.sphinx("latex", testroot="simple")
-def test_simple(tex: str):
-    assert r"\sphinxincludegraphics[]" \
-           r"{drawio-6cf867c6e94665d8489581a35c2d215220d6c152.pdf}" in tex
+def test_simple(tex_images: List[Path]):
+    image, = tex_images
+    assert image.basename() == "drawio-6cf867c6e94665d8489581a35c2d215220d6c152.pdf"
+
+
+@pytest.mark.sphinx("latex", testroot="simple")
+def test_crop(tex_images: List[Path]):
+    image, = tex_images
+    assert get_mediabox(image) == ("88.080002", "46.079998")
```

Comment:
Take a look at `test_options`: https://github.com/Modelmat/sphinxcontrib-drawio/blob/993f534e99b364be01563f0d909b11a6e0ad8d72/tests/test_options.py#L44

I'm assuming that these are the pixel sizes? If so, why are they non-integer?

Reply:
No, these are dimensions in (PostScript) points. The PDF image is a vector format, so no pixel sizes.

Reply:
Okay.